atlas/portal/Makefile

```diff
@@ -0,0 +1,24 @@
+#include Makefile.vars
+#include Makefile.common
+
+
+
+# to override a target from Makefile.common just redefine the target.
+# you can also chain the original atlas target by adding
+# -atlas to the dependency of the redefined target
+
+.PHONY: docker-build-generator
```

Comment:
I'm not sure I understand. Why do we need to add these targets? We have targets that do pretty much the same thing in the Makefile.common: protobuf-atlas for example.

Reply:
protobuf-atlas work only with vendor directory (vendor directory don't use for service), this fact forced me use my own protobuf generator. On the other hand, could use protobuf-atlas with vendor and dont commit vendor dir.